lib/src/cpp/common/LocatorList.hpp

```diff
@@ -21,7 +21,7 @@
 
 #include <string>
 
-#include <xercesc/dom/DOM.hpp>
```

Comment:
I do not think we should remove this header as the `DOMElement` is still passed as parameter to the functions.